spk/homeassistant/Makefile

```diff
@@ -7,20 +7,18 @@ SPK_DEPENDS = "python3>=3.7.7"
 
 BUILD_DEPENDS = cross/python3 cross/setuptools cross/pip cross/wheel
 BUILD_DEPENDS += cross/poetry
-OPTIONAL_DEPENDS += cross/numpy
 
 DEPENDS += cross/libyaml cross/bcrypt cross/cryptography
+
+# for iqvia
+DEPENDS += cross/numpy
+
 DEPENDS += cross/pillow
```

Comment:
pillow is a general dependency and not for iqvia, so I propose to reorder like this:

```suggestion
DEPENDS += cross/libyaml cross/bcrypt cross/cryptography
DEPENDS += cross/pillow
# for iqvia
DEPENDS += cross/numpy

```